src/routes.js

```diff
@@ -42,12 +72,36 @@ function createRouter() {
 
   router.put('/vote', voteHttp.putVote);
 
+  router.post('/feedback/:id', feedbackHttp.postFeedback);
+
   router.get('/radio', radioHttp.getStations);
   router.get('/radio/:id', radioHttp.getStation);
 
   router.put('/mood', wappuMood.putMood);
   router.get('/mood', wappuMood.getMood);
 
+  router.post('/login', requireLogin, loginHttp.login);
+  router.post('/changepassword', requireAuth, loginHttp.changePW);
+  router.post('/activateaccount', requireAuth, loginHttp.activateAccount);
+  router.post('/addmoderator', requireAdmin, loginHttp.addModerator);
+  router.get('/forgottenpassword/:email', loginHttp.forgottenPW);
+  router.put('/promote/:id', requireAdmin, loginHttp.promote);
+  router.put('/demote/:id', requireAdmin, loginHttp.demote);
+  router.put('/admin/users/:id/ban', requireAuth.unless(uuidCheck), adminHttp.shadowBanUser);
+  router.put('/admin/users/:id/unban', requireAuth.unless(uuidCheck), adminHttp.unBanUser);
+  router.delete('/deletemoderator/:id', requireAdmin, loginHttp.deleteModerator);
+  router.get('/moderatorlist', requireAdmin, loginHttp.modList);
+  router.post('/addevent', requireAuth, eventHttp.addEvent);
+  router.delete('/deleteevent/:id', requireAuth, eventHttp.deleteEvent);
+  router.post('/updateevent/:id', requireAuth, eventHttp.updateEvent);
+  router.get('/updateevent/:id', requireAuth, eventHttp.getUpdateEvent);
+  router.post('/admin/actions', requireAuth, adminHttp.sendSystemMessage);
+  router.get('/refreshcommentnumber/:id', feedHttp.refreshCommentNumber);
+
+  router.post('/reports', abuseHttp.reportFeedItem);
+  router.get('/admin/reports', requireAuth, abuseHttp.getReportedFeedItems);
+  router.put('/admin/reports/:id', requireAuth, abuseHttp.resolveReport);
```

Comment:
Generally in REST-like APIs CRUD operations are done via `POST`, `GET`, `PUT` and `DELETE` requests. It's common knowledge, so operation doesn't need to be included in the endpoint. 
Eg. `POST` to `/addevent` should be just `/event`.

Reply:
Requires a bit of fixing with the frontend as well, but will streamline endpoints.